mangaki/mangaki/wrappers/anilist.py

```diff
@@ -277,17 +366,18 @@ def get_user_list(self,
             {'username': username, 'worktype': worktype.value}
         )
 
-        for list_type in data['lists']:
-            for list_entry in data['lists'][list_type]:
-                try:
-                    yield AniListUserWork(
-                        anilist_id=int(list_entry['series_id']),
-                        title=list_entry[worktype.value]['title_romaji'],
-                        poster=list_entry[worktype.value]['image_url_lge'],
-                        score=int(list_entry['score'])
-                    )
-                except KeyError:
-                    raise RuntimeError('Malformed JSON, or AniList changed their API.')
+        if not data:
+            yield None
```

Comment:
It's not a very good idea, you'd have to add `if not item: break` in each usage of this function, we should signal to the user that we have nothing to give: `raise StopIteration` (I think will do the job).
Raising this exception will signal that the generator has nothing to give and will properly convert this `list(user_list)` into a `[]`